rom/laghos_solver.cpp

```diff
@@ -833,6 +832,434 @@ void LagrangianHydroOperator::UpdateQuadratureData(const Vector &S) const
     timer.quad_tstep += nzones;
 }
 
+/// Trace of a square matrix
+template<int H, int W, typename T>
+//MFEM_HOST_DEVICE inline
```

Comment:
Why does this need to be commented out? It seems to work on Mac.

Reply:
The lines were commented in attempt to find a compilation error. The error had been located somewhere else. These lines are uncommented now.